pkg/controller/builder.go

```diff
@@ -91,6 +93,7 @@ func (b *Builder) BuildControllers(subDirectories ...string) (map[string]factory
 					b.Clients.KubeInformerFactory.Apps().V1().Deployments(),
 					nil,
 					[]deploymentcontroller.ManifestHookFunc{
+						replaceVerbosityHook("${LOG_VERBOSITY}"),
```

Comment:
Is this "variable" used anywhere? Is it somewhere in our manifests?

Reply:
It will be. This needs to be here and doing replacements before it shows up in our manifests.